apps/2024/src/app/(main)/application/profile/profile-form.tsx

```diff
@@ -698,7 +698,32 @@ function SchoolsCombobox({
         onValueChange={setSearch}
         placeholder="Search for more schools..."
       />
-      <CommandEmpty>No school found.</CommandEmpty>
+      <CommandEmpty>
+        <div className="mb-2 p-2 text-sm">
+          No school found. Please enter your school below instead.
+          <Input
+            className="t-2"
+            onChange={(e) =>
+              form.setValue(
+                "school",
+                e.target.value
+                  .split(" ")
+                  .map((word) => {
+                    if (
+                      word.toLowerCase() === "of" ||
+                      word.toLowerCase() === "for"
+                    ) {
+                      return word.toLowerCase();
+                    } else {
+                      return word.charAt(0).toUpperCase() + word.slice(1);
+                    }
+                  })
+                  .join(" "),
+              )
+            }
```

Comment:
prob better to use a regex

```
const pattern = /\b(?!for\b|of\b)(\w+)\b/gi;
const result = text.replace(pattern, (match) => match.charAt(0).toUpperCase() + match.slice(1).toLowerCase())
```
can't run locally rn so don't know if this works but shouldn't be far off